src/tests/Microsoft.PowerFx.Interpreter.Tests/ConfigTests.cs

```diff
@@ -0,0 +1,517 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Binding.BindInfo;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Types.Enums;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Intellisense;
+using Microsoft.PowerFx.Tests.IntellisenseTests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class ConfigTests
+    {
+        // Drection evaluation with values. 
+        [Fact]
+        public async Task BasicEvaluation()
+        {
+            // Per expression.
+            var engine = new RecalcEngine();
+            engine.UpdateVariable("global", 2.0);
+
+            var locals = new SymbolValues()
+                .AddVariable("local", FormulaValue.New(3));
```

Comment:
UpdateVariable vs. AddVariable
Shouldn't we use the same name?

Reply:
I'll rename to just Add(). 

Add existing should fail. 

Engine.UpdateVariable is existing method.